python/dgl/graphbolt/graph_storage/csc_sampling_graph.py

```diff
@@ -304,6 +304,59 @@ def sample_neighbors(
             nodes, fanouts.tolist(), replace, return_eids, probs_or_mask
         )
 
+    def sample_negative_edges_uniform(
+        self, node_pairs, negative_ratio, edge_type
```

Comment:
Let's order the parameters in a more natural way. edge_type decides how to understand node_pairs, it should group with node_pairs, and negative_ratio indicates the operator details.
```suggestion
        self, edge_type, node_pairs, negative_ratio
```

Reply:
Make sense, changed for fix